corehq/apps/api/resources/v0_5.py

```diff
@@ -253,6 +254,8 @@ def obj_create(self, bundle, request=None, **kwargs):
                 domain=kwargs['domain'],
                 username=bundle.data['username'].lower(),
                 password=bundle.data['password'],
+                created_by=bundle.request.couch_user,
```

Comment:
Here you're using `request.couch_user` but below you are using `request.user`. Is that intentional?

Reply:
thanks for catching that, missed updating that instance to use request.user now

Reply:
Both couch user and django user are supported, that was intentional. And then where ever we have User available, we should pass that itself.

Reply:
https://github.com/dimagi/commcare-hq/pull/27755/commits/4b3f3fbaffbca5a0f992c74f52ad242a82de4526